splash/render_options.py

```diff
@@ -20,14 +20,15 @@ def __init__(self, data):
         self.data = data
 
     @classmethod
-    def fromrequest(cls, request):
+    def fromrequest(cls, request, max_timeout=defaults.MAX_TIMEOUT):
```

Comment:
I think it is cleaner to provide a `get_max_timeout` method - other options are not passed to the fromrequest method.